examples/zone/main.go

```diff
@@ -0,0 +1,58 @@
+// Calls the zone endpoint and outputs the result
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/simplifi/ultradns-go/pkg/ultradns"
+)
+
+func main() {
+	// Use a couple of flags to allow passing in the username/password.
+	userPtr := flag.String("user", "", "Username for UltraDNS API")
+	passPtr := flag.String("pass", "", "Password for UltraDNS API")
+	zonePtr := flag.String("zone", "", "Zone to view")
+
+	flag.Parse()
+
+	if *userPtr == "" || *passPtr == "" {
+		flag.PrintDefaults()
+		return
+	}
+
+	// Create an APIConnection with the username/password provided.
+	apiConn := ultradns.NewAPIConnection(&ultradns.APIOptions{
+		Username: *userPtr,
+		Password: *passPtr,
+	})
+
+	var url string
+
+	if *zonePtr == "" {
+		url = "/zones/"
+	} else {
+		url = "/zones/" + *zonePtr
+	}
```

Comment:
Does go get mad if you concatenate `""`?  If not then can we just do `url = "/zones/" + *zonePtr` without the conditional?

Reply:
oh hell, this is just an example.  Never mind unless you're in a code-golfing mood.

Reply:
Good call

Reply:
fixed